.github/workflows/build.yml

```diff
@@ -14,17 +14,10 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest]
-        # os: [ubuntu-latest, windows-latest, macos-latest]
```

Comment:
let's keep this, since the toolchains are different. Maybe for windows + MAC we only run generic.h and tdongles3, not all the other boards.

Reply:
I will do that

Reply:
see https://github.com/t-huyeng/ESP32-Paxcounter/actions/runs/3395574336  for the runs.

Is this okay @cyberman54 ?

Reply:
I also added the `ttgov21new.h` since this is probably the default board for a lot of people.

Reply:
perfect! Thanks.